docs/release/release-tasks.md

```diff
@@ -483,13 +483,15 @@ The goal of this task is to keep our tests running in CI stable.
     (also dashboards of previous releases). Using the alert mailing list has proven to be a lot less effort though.
 2. Subscribe to `CI Activity` notifications for the Cluster API repo.
 3. Check the existing **failing-test** and **flaking-test** issue templates under `.github/ISSUE_TEMPLATE/` folder of the repo, used to create an issue for failing or flaking tests respectively. Please make sure they are up-to-date and if not, send a PR to update or improve them.
-4. Triage CI failures reported by mail alerts or found by monitoring the testgrid dashboards:
+4. Check if there are any existing jobs that got stuck in ['pending'](https://prow.k8s.io/?repo=kubernetes-sigs%2Fcluster-api&state=pending) state:
```

Comment:
Would be good to add a note here that if a job has been running for more than 12 hours it can be considered "stuck". Actually it's ~2 hours, but a healthy buffer is fine and should avoid over-reporting of this.

Reply:
Thanks for context, clarified it now.